src/main/scala/uk/gov/hmrc/ct/accounts/frs102/boxes/AC13.scala

```diff
@@ -28,9 +28,15 @@ case class AC13(value: Option[Int]) extends CtBoxIdentifier(name = "Turnover (pr
   with Validators {
 
   override def validate(boxRetriever: FullAccountsBoxRetriever): Set[CtValidation] = {
-    collectErrors(
+    val errors = collectErrors(
       validateInputAllowed("AC13", boxRetriever.ac205()),
-      validateMoney(value)
+      validateZeroOrPositiveInteger(this)
     )
+
+    if(errors.isEmpty) {
+      validateMoney(value, 0)
```

Comment:
I suppose it just comes down to what error message we want to serve. the validate money error is something like "enter a figure between £0 and £9999999" while the non-negative would say "so and so cannot be negative"